default/python/AI/FreeOrionAI.py

```diff
@@ -3,12 +3,16 @@
 import pickle  # Python object serialization library
 import sys
 import random
+import logging
 
-from common.configure_logging import redirect_logging_to_freeorion_logger
-redirect_logging_to_freeorion_logger()
+from common.configure_logging import redirect_logging_to_freeorion_logger, convenience_function_references_for_logger
 
 import freeOrionAIInterface as fo  # interface used to interact with FreeOrion AI client  # pylint: disable=import-error
 
+log_level = logging.DEBUG if fo.getOptionsDBOptionBool("verbose-logging") else logging.INFO
```

Comment:
Does not seem to be routed to the intended channel?

Results of AI_1.log for the `initialized` line + next `: : :`

--log-level INFO --verbose-logging 0
```
2017-02-10 16:55:42.294527 [info] AI : : :  configure_logging.py:redirect_logging_to_freeorion_logger():199  - The python logger is initialized with a log level of INFO
2017-02-10 16:55:45.159839 [info] AI : : :  FreeOrionAI.py:generateOrders():211  - Meter / Resource Pool updating...
```

--log-level INFO --verbose-logging 1
```
2017-02-10 16:56:53.941233 [info] AI : : :  configure_logging.py:redirect_logging_to_freeorion_logger():199  - The python logger is initialized with a log level of DEBUG
2017-02-10 16:56:56.832946 [info] AI : : :  FreeOrionAI.py:generateOrders():211  - Meter / Resource Pool updating...
```

--log-level DEBUG --verbose-logging 0
```
2017-02-10 16:57:50.251641 [info] AI : : :  configure_logging.py:redirect_logging_to_freeorion_logger():199  - The python logger is initialized with a log level of INFO
...
2017-02-10 16:57:53.703864 [debug] AI : : : ((4.0, 1), 15, 0, 0, 0, 0)
```

--log-level DEBUG --verbose-logging 1
```
2017-02-10 17:01:07.298026 [info] AI : : :  configure_logging.py:redirect_logging_to_freeorion_logger():199  - The python logger is initialized with a log level of DEBUG
2017-02-10 17:01:07.480485 [debug] AI : : : FreeOrionAI.py:<module>():54  - Path to folder for user specific data: /home/cxdamian/.local/share/freeorion
```

Not certain what the full intent here is, hopefully it is not to override the `log-level` config setting.